llpc/test/shaderdb/partial_pipeline_cache_test/pipelines/PipelineVsFs_test1_Vs1Fs1WithData.pipe

```diff
@@ -0,0 +1,106 @@
+; Test that constant data in the vertex shader is handled correctly.
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -enable-relocatable-shader-elf -o %t.elf %gfxip %s && llvm-objdump --arch=amdgcn --disassemble-zeroes --mcpu=gfx900 -D -r %t.elf | FileCheck -check-prefix=SHADERTEST %s
```

Comment:
This line is very long. Can you break it into multiple shorter ones?
```
; RUN: amdllpc -spvgen-dir=%spvgendir% -enable-relocatable-shader-elf -o %t.elf %gfxip %s
; RUN: llvm-objdump --arch=amdgcn --disassemble-zeroes --mcpu=gfx900 -D -r %t.elf | FileCheck -check-prefix=SHADERTEST %s
```

Reply:
Sure.